src/met-atlas-viewer.js

```diff
@@ -158,6 +158,10 @@ function MetAtlasViewer(targetElement) {
 
   let showGenes = true;
 
+
+  // animation id to be used when disposing scene
+  let animationId
```

Comment:
I think a linter would complain about a missing semicolon here.

Reply:
Good catch. Admittedly I wasn't too concerned with it when working on this current issue. I also notice that there are some tabs (should probably have a rule for no tabs allowed imo) sprinkled here and there in some files. I could fix this along with other changes if there are more fixes needed, but in principle I would prefer to handle this in a separate issue, what do you think?

Reply:
Sounds good - the use of a linter is now a separate issue #34.

Reply:
Great, thanks!